### ISSUE TITLE
Possible to create infinite empty paragraph blocks at end of editor

### ISSUE THREAD

**Issue by mrwweb** [WordPress/gutenberg#56401]
### Description

As of WordPress 6.4, it seems that one can continually create new empty paragraph blocks at the end of the post by clicking below the last paragraph.

### Step-by-step reproduction instructions

1. Make a new post
2. Place cursor in the last empty paragraph block
3. Click below the block
4. A new paragraph block with no placeholder text appears
5. Repeat Steps 2-4 forever

### Screenshots, screen recording, code snippet

This video demonstrates the step-by-step reproduction:

https://github.com/WordPress/gutenberg/assets/1238696/621482af-7b23-465a-838e-710f11c7026a

### Environment info

- WordPress 6.4
- Multiple custom themes tested with same result
- Observed in Windows Firefox and MacOS Chrome

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

No

**Comment by annezazu** [comment]
@jasmussen this might be related to your PR that removed the placeholder. Can you speak to this at all?!

**Comment by jasmussen** [comment]
I don't know that I'd call this a bug, since it's arguably how editors should work, with Enter creating linebreaks. This wouldn't be substantially different by repeating the "Type /" prompt down the page. My personal opinion is that this is working as intended, but I can see how a bunch of empty paragraphs at the end, which show up only in the list view, could be perceived as bad markup. How would you feel about a "linting" type interface, perhaps one that shows up in the pre-publish flow? Such an interface has already been discussed in terms of links that link to `#`, or buttons that link nowhere.

**Comment by mrwweb** [comment]
tl;dr - The bug report is specifically about empty paragraphs unexpectedly generated by clicking in empty space. There were additional parts of the experience that made it _feel_ even more like a bug.

@jasmussen I think the title of my bug report is poorly worded in retrospect and didn't really capture what I was trying to communicate. I'll try again!

1. The primary issue is that **clicking** in the empty space below the final paragraph creates a new paragraph. There's nothing to indicate an interaction is possible there, and I think the first time it happened was because I was trying to click the last paragraph and missed. At a minimum, it really _feels_ like a bug.
2. The fact that the first instance of the empty paragraph and the subsequent ones don't also feels broken, even if it's intentional. I can understand the rationale, but it felt weird since I know there's no difference between them under the hood. And not having the placeholder makes clicking into any paragraph without it harder.

> "since it's arguably how editors should work, with Enter creating linebreaks."

3. I hear that, and I spent a while thinking about it before responding. First, I would say that if you want to keep that behavior, the clicking-in-empty-space-makes-empty-paragraphs thing (point 1 above, shown in video) is not standard behavior and still feels like a bug. Beyond that, I'm not convinced it's a good idea at all. The impact of empty paragraphs in a word processor is empty space. That's often [not the case in HTML](https://codepen.io/mrwweb/full/ZEwMWGM). At least in my experience, allowing lots of empty paragraphs is one of the easiest ways to make the editor design not look like the front-end design.

**Comment by jasmussen** [comment]
> The bug report is specifically about empty paragraphs unexpectedly generated by clicking in empty space. There were additional parts of the experience that made it feel even more like a bug.

Apologies. I've had some busy days, thank you for clarifying. Agreed, that isn't helpful. Consecutive presses of Enter feels intentional, but just clicking (with the mouse, below the content) does not seem like an editorial action. 👍 👍

**Comment by kartikmehta8** [comment]
Hi @jasmussen! We can close this as this is fixed. 👍🏻
